drivers/serial/Kconfig.imx

```diff
@@ -0,0 +1,73 @@
+# Kconfig - iMX7 M4 UART
```

Comment:
iMX6SX uses the same driver in FreeRTOS_BSP_1.0.1_iMX6SX. Not sure whether to change the iMX7 mentions to something generic. Or I can mention the other SoC once I add its support.

Reply:
In my opinion, all the generalization should be put in place when a new platform is going to be added.

Reply:
Ok.